cmd/defaults.yaml

```diff
@@ -1321,6 +1321,9 @@ InternalAuthZ:
         - "userschema.read"
         - "userschema.write"
         - "userschema.delete"
+        - "session.read"
+        - "session.link"
```

Comment:
the initial idea was to not allow IAM_OWNER's the permission to link sessions as then even human iam owners would be allowed to create oidc sessions for others
but i guess you would anyway need a session token to do so, so i guess it's no additional risk

Reply:
I am fine with removing, I guess the session read would be usefull, if we want to provide a page in where you can see open sessions for different users, maybe also for support request. but create/link is probably not necessary.

Reply:
i removed the link for now but totally agree on the read and also delete functionality for iam owners